internal/cli/atlas/deployments/setup.go

```diff
@@ -769,6 +771,8 @@ func SetupBuilder() *cobra.Command {
 
 	// Local only
 	cmd.Flags().IntVar(&opts.Port, flag.Port, 0, usage.MongodPort)
+	cmd.Flags().BoolVar(&opts.bindAllIPs, flag.BindAllIPs, false, usage.BindAllIPs)
+	_ = cmd.Flags().MarkHidden(flag.BindAllIPs)
```

Comment:
Q: Is this hidden for an initial period or would it be always hidden?

Reply:
change it to visible in the meantime, as auth support was implemented here